torch/csrc/lazy/core/dynamic_ir.cpp

```diff
@@ -11,6 +11,19 @@ std::string DimensionNode::ToString() const {
   return "DimensionNode";
 }
 
+  TSOpVector SizeNode::Lower(std::shared_ptr<torch::jit::GraphFunction> function,
+                          TSLoweringContext* loctx) const {
+  std::vector<torch::jit::NamedValue> arguments;
+  std::vector<torch::jit::NamedValue> kwarguments;
+  arguments.reserve(2);
+  auto index = loctx->graph()->insertConstant(static_cast<int64_t>(this->dim_));
+  arguments.emplace_back(loctx->GetOutputOp(operand(0)));
```

Comment:
i guess this works for all cases (where operand(0) is a constant, a list of constants, a Value, etc.?)

it's cool to see this E2E finally!

Reply:
Err I think operand(0) should be a tensor otherwise JIT would burp.